code/obj/item/Tumbling_Creeper_trap.dm

```diff
@@ -0,0 +1,334 @@
+	flags = TABLEPASS | FPRINT | NOSPLASH
+	w_class = W_CLASS_NORMAL
+	force = 3
+	throwforce = 0
+	stamina_damage = 0
+	stamina_cost = 0
+	stamina_crit_chance = 5
+	var/datum/plant/planttype = /datum/plant/weed/creeper
+	var/datum/plantgenes/plantgenes = null
+	var/generation = 0 // For genetics tracking.
+	var/armed = FALSE //! This determinates if the trap is armed or not
+	var/endurance_for_max = 100 //! how much endurance is needed to reach max damage with the trap
+	var/potency_for_max = 200 //! how much potency is needed to generate the max injection-multiplier
+	var/armed_force = 8 //! how much damage the trap does when stepped upon. Will be set when harvested
+	var/armed_force_min = 8 //! how much damage the trap does when stepped upon with 0 endurance
+	var/armed_force_max = 14 //! how much damage the trap does when stepped upon with the maximum endurance
+	var/armed_weakened = 2 SECONDS //! how long you are weakened after stepping into the trap
+	var/crashed_force = 20 //! how much damage the trap does when crashed into. Will be set when harvested
+	var/crashed_force_min = 20 //! how much damage the trap does when stepped upon with 0 endurance
+	var/crashed_force_max = 30 //! how much damage the trap does when stepped upon with the maximum endurance
+	var/crashed_weakened = 3 SECONDS //! how long you are stunned if you crash into the trap
+	var/reagent_storage = 8 //! How much the max amount of chems is the trap should be able to hold
+	var/reagent_storage_min = 8 //! How much the max amount of chems is the trap should be able to hold	at 0 potency
+	var/reagent_storage_max = 50 //! How much the max amount of chems is the trap should be able to hold	at max potency
+	var/reagent_generation_multiplier = 0.5 //! How much percentage of the volume should be filled with assoc_reagents when harvested
+	var/stepon_transfer_multiplier = 0.5 //! Multiplier to damage to calculate the amount of chems tranferred when stepped into.
+	var/crash_transfer_multiplier = 0.4 //! Multiplier to damage to calculate the amount of chems tranferred when crashed into.
+	var/target_zone = "chest" //! which zone the trap tries to target and calculate the damage resist from
+	var/disarming_time = 3 SECONDS //! how long disarming with a wirecutter should take
+	var/arming_time = 2 SECONDS //! how long arming should take
+	var/tumbling_cooldown = 15 SECONDS //!how long the item should take at minimum before it begins tumbling again
+	var/tumbling_distance_max = 6 //! how many tiles the item tries to move at most when tumbling
+	var/tumbling_distance_min = 3 //! how many tiles the item tries to move at least when tumbling
+	var/tumbling_speed = 0.3 //!how fast the throw while tumbling should be
+	var/tumbling_flip_duration = 1.25 SECONDS //!how long a flip of the tumbler should take
+	var/tumbling_chance = 20 //! the chance in % the item tries to thumble on each process tick
+	var/on_Ground = TRUE //! changed whenever the item is picked up or dropped
```

Comment:
capitals.

Reply:
huh, i forgot to delete that variable anyway, since i solved the use i had for this differently...